sources/packages/web/src/components/common/restriction/ViewRestriction.vue

```diff
@@ -3,7 +3,7 @@
     <modal-dialog-base title="View restriction" :showDialog="showDialog">
       <template #content>
         <error-summary :errors="viewRestrictionForm.errors" />
-        <h3 class="category-header-medium mb-5">Restriction information</h3>
+        <h3 class="category-header-medium mb-2">Restriction information</h3>
```

Comment:
Can we use the same class to add margins in restriction, resolution and deletion area?